packages/manager/modules/pci/src/projects/project/storages/databases/features.constants.js

```diff
@@ -0,0 +1,36 @@
+import includes from 'lodash/includes';
+import { DATABASE_TYPES } from './databases.constants';
+
+const features = {
+  databaseTab: [DATABASE_TYPES.MYSQL, DATABASE_TYPES.POSTGRESQL],
+  certificate: [
+    DATABASE_TYPES.MYSQL,
+    DATABASE_TYPES.POSTGRESQL,
+    DATABASE_TYPES.KAFKA,
+  ],
+  replicateURI: [DATABASE_TYPES.MONGO_DB],
+  usersTab: [
+    DATABASE_TYPES.MONGO_DB,
+    DATABASE_TYPES.MYSQL,
+    DATABASE_TYPES.POSTGRESQL,
+    DATABASE_TYPES.KAFKA,
+    DATABASE_TYPES.REDIS,
+  ],
+  backupTab: [
+    DATABASE_TYPES.MONGO_DB,
+    DATABASE_TYPES.MYSQL,
+    DATABASE_TYPES.POSTGRESQL,
+    DATABASE_TYPES.REDIS,
+  ],
+  restoreFromBackup: [DATABASE_TYPES.MONGO_DB],
+  aclTab: [DATABASE_TYPES.KAFKA],
+  topicsTab: [DATABASE_TYPES.KAFKA],
+  getRoles: [DATABASE_TYPES.MONGO_DB, DATABASE_TYPES.POSTGRESQL],
+  showKey: [DATABASE_TYPES.KAFKA],
+  showCert: [DATABASE_TYPES.KAFKA],
+  showUserInformations: [DATABASE_TYPES.REDIS],
+};
+
+export default function isFeatureActivated(feature, databaseType) {
+  return includes(features[feature], databaseType);
```

Comment:
featureAvailability 2API is used activate/deactivate features. It supports feature activation/deactivation dynamically based on region, nic, etc. Please take a look at it.